### DISCUSSION TITLE
Welcome to pigallery2 Discussions!

### DISCUSSION THREAD

**Discussion by bpatrik** [bpatrik/pigallery2#210] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * If you are missing a feature, see https://github.com/bpatrik/pigallery2#4-suggestendorse-new-features.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robertugo2** [comment]
Hi. I think I have a problem with folder filtering... I'm trying to exclude folders with space inside, like "some folder name". When I'm putting this folder name in "Exclude Folder List", even inside ' or ", all white spaces are replaced with ;. I'm doing something wrong or it is expected behaviour?

**Comment by kleinrene** [comment]
Hello,

where is a manuel to upload marked faces from lightroom or orther tools?

Thx for your very nice project!

BG kleinrene

**Reply by bpatrik** [reply]
Hi,

This app is directory based. You use whatever tool you like to edit your photos and save them to you wherever you like. Then you just point the app to that folder and it works.
